src/components/timeSabarCounter/saberTimeStruct.js

```diff
@@ -0,0 +1,36 @@
+import './saberTime.css';
+
+function createHTML() {
+  const divOuter = document.createElement('div');
+  divOuter.id = 'lightSaberProgresBar';
+
+  const image = document.createElement('img');
+  image.src = './images/sabre.png';
```

Comment:
I guess this doesn't have to be a relative link. Just `images/sabre.png` should do.

Reply:
It's changed